packages/iTwinUI-react/src/core/utils/hooks/useDragAndDrop.tsx

```diff
@@ -0,0 +1,68 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Decided to move drag logic to a separate hook if we need it in the future and it was nice to use it inside dialog.